projects/packages/jetpack-mu-wpcom/src/features/starter-page-templates/page-patterns-plugin.tsx

```diff
@@ -75,15 +78,7 @@ export function PagePatternsPlugin( props: PagePatternsPluginProps ): JSX.Elemen
 		};
 	}, [] );
 
-	const { getMeta, postContentBlock } = useSelect( select => {
-		const getMetaNew = () =>
-			( select( 'core/editor' ) as CoreEditorPlaceholder ).getEditedPostAttribute( 'meta' );
-		const currentBlocks = ( select( 'core/editor' ) as CoreEditorPlaceholder ).getBlocks();
-		return {
-			getMeta: getMetaNew,
-			postContentBlock: findPostContentBlock( currentBlocks ),
-		};
-	}, [] );
+	const { getMeta, postContentBlock } = selectProps;
```

Comment:
Why not deconstruct from `useSelect`?

Reply:
No reason. It's what the code already did, and I wanted to keep the changes minimal.